tim_sort/Python/tim_sort.py

```diff
@@ -0,0 +1,13 @@
+
+def tim_sort(arr):  # Function that performs the tim sort
+   arr.sort()
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpc083vmnv/tim_sort/Python/tim_sort.py
+++ b/tmp/tmpc083vmnv/tim_sort/Python/tim_sort.py
@@ -1,6 +1,6 @@
 
 def tim_sort(arr):  # Function that performs the tim sort
-   arr.sort()
+    arr.sort()
 
 
 def main():
```